packages/trusted-contribution/package.json

```diff
@@ -27,9 +27,10 @@
     "lint": "gts check"
   },
   "dependencies": {
-    "gcf-utils": "^5.3.0"
+    "gcf-utils": "^6.1.1"
   },
   "devDependencies": {
+    "@probot/octokit-plugin-config": "^1.0.0",
```

Comment:
I'm curious - are we 100% sure this is a dev dependency and not required as a regular dependency?

Reply:
This plugin is added in `gcf-utils`, so exists on the octokit instances passed to functions in production.

We could potentially expose the Octokit instance uses in `gcf-utils`.